mongo-dao/src/main/resourses/application.properties

```diff
@@ -1,2 +1,2 @@
-spring.data.mongodb.uri= mongodb+srv://Arundhathy:Ammu2002@kovan.us1gd.mongodb.net/?retryWrites=true&w=majority&appName=Kovan
-spring.data.mongodb.database = NewsDB
+spring.config.import=aws-parameterstore:/config/NewsService/
+spring.data.mongodb.uri= ${MongoDbURI}
```

Comment:
Is this spring.data.mongodb.uri= ${MongoDbURI} needed ?

Reply:
spring.config.import=aws-parameterstore:/config/NewsService/ and
spring.data.mongodb.uri= ${MongoDbURI} Removed.